frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -56,15 +65,36 @@ export const BrowseModels = ({
 
   if (modelsFiltered.length) {
     return (
-      <GridContainer role="grid">
-        {groupsOfModels.map(groupOfModels => (
-          <ModelGroup
-            models={groupOfModels}
-            key={`modelgroup-${groupOfModels[0].collection.id}`}
-            localeCode={localeCode}
-          />
-        ))}
-      </GridContainer>
+      <>
+        {shouldShowBanner && (
+          <Banner>
+            <BannerModelIcon>
+              <Icon name="model" />
+            </BannerModelIcon>
+            <Text size="md" lh="1rem" mr="1rem">
+              Models help curate data to make it easier to find answers to
+              questions all in one place.
+            </Text>
+            <BannerCloseButton
+              onClick={() => {
+                updateSetting({ key: "dismissed-model-banner", value: true });
```

Comment:
This action needs to be dispatched. You'll either need to use the redux connect api or preferably the `useDispatch` hook. By doing this you also won't need forceUpdate

Reply:
It's also generally good practice to throw things like these inside of useCallbacks. Doesn't matter much here, but it can in other situations